widget/entry_test.go

```diff
@@ -57,6 +57,36 @@ func TestMultiLineEntry_MinSize(t *testing.T) {
 	assert.Equal(t, singleMin.Height, multiMin.Height)
 }
 
+func TestMultiLineEntry_Scroller_MinSize(t *testing.T) {
+	text := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	size := textMinSize(text, theme.TextSize(), fyne.TextStyle{})
+
+	multi := NewMultiLineEntry()
+	multi.SetText(text)
+	multiMin := multi.MinSize()
+	assert.Equal(t, size.Width+2*theme.Padding()+theme.ScrollBarSize(), multiMin.Width)
+	assert.Equal(t, size.Height*multiLineRows+2*theme.Padding(), multiMin.Height)
+
+	t.Run("Both", func(t *testing.T) {
+		scroller := NewScrollContainer(multi)
+		scrollerMin := scroller.MinSize()
+		assert.Equal(t, 32, scrollerMin.Height)
+		assert.Equal(t, 32, scrollerMin.Width)
+	})
+	t.Run("HorizontalOnly", func(t *testing.T) {
+		scroller := NewHorizontalScrollContainer(multi)
+		scrollerMin := scroller.MinSize()
+		assert.Equal(t, size.Height*multiLineRows+2*theme.Padding(), scrollerMin.Height)
+		assert.Equal(t, 32, scrollerMin.Width)
+	})
+	t.Run("VerticalOnly", func(t *testing.T) {
+		scroller := NewVerticalScrollContainer(multi)
+		scrollerMin := scroller.MinSize()
+		assert.Equal(t, 32, scrollerMin.Height) // TODO FIXME Should equal size.Height*multiLineRows+2*theme.Padding()
```

Comment:
@andydotxyz @toaster can I draw you attention to this test which shows scrollers do not respect multiline entries multiLineRows? If a height of 32 is okay then I can remove the comment, but if we want multiline entries to always display at least multiLineRows then there will need to be more coordination between entries and scrollers.

Reply:
Absolutely. The conversation about how an entry should be presented, and how it should scroll is ongoing, you can see it in the discussion about test wrapping and the issue #471.
The minSize for entry in multiline mode is correct - but when you put it in a scroller that overrides the minSize. Scrollers do not know anything about their content, and so could not query the Entry.multilineRows property...
Surely this is a separate issue to the discussion in hand?

Reply:
You're right - this is a separate issue to limiting scroll directions. For now I'll leave the test as is (with comment) and we can come back to it when we have an answer for #471

Reply:
Sorry somehow I missed that this was new tests. I’d be tempted to say don’t add entry scroll tests just now it’s a whole other topic that belongs in one of the other discussions. Could this perhaps be added as a patch attachment to an appropriate discussion on entry and left out of this PR for now? 

Thanks for bringing it to your attention, sorry I didn’t notice the addition of new entry tests.

Reply:
Reverted